### PR TITLE
chore(plus): prepare account rebranding

### PR DESCRIPTION
**Pull request by fiji-flo**
## Summary

Preparing our accounts rebranding

### CODE REVIEW

File: client/src/settings/manage.tsx [hunk 1]
```diff
@@ -69,15 +69,15 @@ export function Manage() {
         )}
 
         <li>
-          <h3>Firefox account</h3>
-          <span>Manage your Firefox account</span>
+          <h3>Your account</h3>
```
Comment (caugner) [inline client/src/settings/manage.tsx:72]: Why don't we just call it "Account"? We don't call it "Your subscription" either.

```suggestion
          <h3>Account</h3>
```

cc @s-sood

File: client/src/settings/manage.tsx [hunk 2]
```diff
@@ -69,15 +69,15 @@ export function Manage() {
...
+          <span>Manage your account</span>
           <a
             rel="noreferrer noopener"
             target="_blank"
             href={FXA_SETTINGS_URL}
             className="manage external"
           >
-            Firefox account
+            Your Account
```
Comment (caugner) [inline client/src/settings/manage.tsx:80]: dito 
```suggestion
            Account
```

**Review by caugner — APPROVED** [review]
LGTM, just a wording question.

File: client/src/settings/manage.tsx (hunk 1 above)
Comment (s-sood) [reply client/src/settings/manage.tsx:72]: Let's just do "Account"

File: client/src/settings/manage.tsx (hunk 2 above)
Comment (s-sood) [reply client/src/settings/manage.tsx:80]: Same, "Account" is also okay

**Pull request merged**